### ISSUE TITLE
Error while adding image to trip using Image object

### ISSUE THREAD

**Issue by clicktravel-tajinder** [tripit/api#73]
Hi, 

We are using the Java Binding for TripIt API to exchange data. 
We are trying to add an Image to an ActivityObject. 
The Image object is as follows:

``` html
<Image>
        <caption>066R1</caption>
        <url>http://api.holidayextras.co.uk/sandbox/v1/barcode/066R1?key=clicktravel</url>
    </Image>
```

An error is returned from TripIt:

``` html
RESPONSE BODY: <Response><Error><code>400</code><description>unable to create new image from http://api.holidayextras.co.uk/sandbox/v1/barcode/066R1?key=clicktravel</description></Error></Response>
```

Please let us know what needs to be done.

Thanks,
Tajinder

**Comment by tariqislam** [comment]
Hi Tajinder,

Your url "http://api.holidayextras.co.uk/sandbox/v1/barcode/066R1?key=clicktravel" is not an image, if you open it in the browser. Or at least, from the open web, I am unable to access the image in that url.

Thanks.

Tariq

**Comment by clicktravel-tajinder** [comment]
Hi Tariq,

Thanks. I have resolved the issue. It was to do with the missing extension from the url. It image url was generated this way and works if specified in the src attribute of <img tag. "http://api.holidayextras.co.uk/sandbox/v1/barcode/066R1?key=clicktravel" wont work in a browser but "http://api.holidayextras.co.uk/sandbox/v1/barcode/066R1.png?key=clicktravel" will work.

I have made the changes.

Thanks,
Tajinder
